lib/activerecord-bitemporal/scope.rb

```diff
@@ -407,15 +407,14 @@ def _#{column}_#{op}(datetime,**)
       }
 
       scope :bitemporal_at, -> (datetime) {
-        datetime = Time.current if datetime.nil?
         relation = self
 
         if !ActiveRecord::Bitemporal.ignore_transaction_datetime?
-          relation = relation.transaction_at(ActiveRecord::Bitemporal.transaction_datetime || datetime)
+          relation = relation.transaction_at(datetime || ActiveRecord::Bitemporal.transaction_datetime || Time.current)
         end
 
         if !ActiveRecord::Bitemporal.ignore_valid_datetime?
-          relation = relation.valid_at(ActiveRecord::Bitemporal.valid_datetime || datetime)
+          relation = relation.valid_at(datetime || ActiveRecord::Bitemporal.valid_datetime || Time.current)
```

Comment:
Calling `Time.current` on valid time and transaction time respectively can give different results in milliseconds. Is it better to use the same time? e.g.

```ruby
current_time = Time.current

if !ActiveRecord::Bitemporal.ignore_transaction_datetime?
  relation = relation.transaction_at(datetime || ActiveRecord::Bitemporal.transaction_datetime || current_time)
end

if !ActiveRecord::Bitemporal.ignore_valid_datetime?
  relation = relation.valid_at(datetime || ActiveRecord::Bitemporal.valid_datetime || current_time)
end
```

Reply:
@wata727 
thank you.
Fixed as suggested.